bftengine/src/bftengine/PersistentStorageImp.hpp

```diff
@@ -99,14 +99,18 @@ enum DescMetadataParameterIds {
   LAST_NEW_VIEW_DESC
 };
 
+const uint32_t clientsDataDescNum = LAST_NEW_VIEW_DESC + 1;
```

Comment:
Define enum instead for better readability.

Reply:
fixed